gulp/scripts.js

```diff
@@ -2,11 +2,17 @@
 
 var gulp = require('gulp');
 var browserSync = require('browser-sync');
+var gulpNgConfig = require('gulp-ng-config');
 
 var $ = require('gulp-load-plugins')();
 
 module.exports = function(options) {
   gulp.task('scripts', function () {
+    gulp.src('configFile.json')
+      .pipe(gulpNgConfig('uvpBuilderWeb.config', {
+        environment: 'local'
```

Comment:
Ideas on how to indicate the environment to gulp when a task is called? @alvarezloaiciga @gsotoruiz @BrayanZ